### ISSUE TITLE
Doesn't work in Python 3.11

### ISSUE THREAD

**Issue by PalmtopTiger** [tkarabela/pysubs2#56]
```
> pysubs2.exe

Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Python311\Scripts\pysubs2.exe\__main__.py", line 4, in <module>
  File "C:\Python311\Lib\site-packages\pysubs2\__init__.py", line 1, in <module>
    from .ssafile import SSAFile
  File "C:\Python311\Lib\site-packages\pysubs2\ssafile.py", line 13, in <module>
    from .formats import autodetect_format, get_format_class, get_format_identifier
  File "C:\Python311\Lib\site-packages\pysubs2\formats.py", line 4, in <module>
    from .microdvd import MicroDVDFormat
  File "C:\Python311\Lib\site-packages\pysubs2\microdvd.py", line 5, in <module>
    from .ssastyle import SSAStyle
  File "C:\Python311\Lib\site-packages\pysubs2\ssastyle.py", line 7, in <module>
    @dataclasses.dataclass(repr=False)
     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\dataclasses.py", line 1211, in wrap
    return _process_class(cls, init, repr, eq, order, unsafe_hash,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\dataclasses.py", line 959, in _process_class
    cls_fields.append(_get_field(cls, name, type, kw_only))
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\dataclasses.py", line 816, in _get_field
    raise ValueError(f'mutable default {type(f.default)} for field '
ValueError: mutable default <class 'pysubs2.common.Color'> for field primarycolor is not allowed: use default_factory
```

**Comment by tkarabela** [comment]
Thanks for the report! I see what the issue is, will fix.I don't yet have Python 3.11 in CI, I'll add that as well.

**Comment by tkarabela** [comment]
I've released version 1.4.4 of the library where Python 3.11 is supported. Please upgrade :)

**Comment by PalmtopTiger** [comment]
Thank you!
